VCL/WavFile.pas

```diff
@@ -16,7 +19,7 @@ interface
 
 uses
   Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
-  MMSystem, SndTypes;
+  MMSystem, SndTypes, PermHint; //grg PermHint added

```

Comment:
PermHint is a visual component, there is no need to use it in the non-visual WavFile.